core/trino-main/src/main/java/io/trino/sql/planner/planprinter/NodeRepresentation.java

```diff
@@ -151,14 +152,14 @@ public Optional<PlanNodeStatsAndCostSummary> getReorderJoinStatsAndCost()
     public List<PlanNodeStatsAndCostSummary> getEstimates(TypeProvider typeProvider)
     {
         if (getEstimatedStats().stream().allMatch(PlanNodeStatsEstimate::isOutputRowCountUnknown) &&
-                getEstimatedCost().stream().allMatch(c -> c.equals(PlanCostEstimate.unknown()))) {
+                getEstimatedCost().stream().allMatch(c -> c.getRootNodeLocalCostEstimate().equals(LocalCostEstimate.unknown()))) {
             return ImmutableList.of();
         }
 
         ImmutableList.Builder<PlanNodeStatsAndCostSummary> estimates = ImmutableList.builder();
         for (int i = 0; i < getEstimatedStats().size(); i++) {
             PlanNodeStatsEstimate stats = getEstimatedStats().get(i);
-            PlanCostEstimate cost = getEstimatedCost().get(i);
+            LocalCostEstimate cost = getEstimatedCost().get(i).getRootNodeLocalCostEstimate();
```

Comment:
Why this called `rootNodeLocalCostEstimate` instead of `nodeLocalCostEstimate` ? Root node makes it sound like we are talking about the root PlanNode of a plan fragment, but here it looks like we obtained the local cost of a particular PlanNode from it.

Reply:
yes. The field name is misleading. Can be a separate PR